fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/stockmedia/SearchStockMediaResponse.kt

```diff
@@ -64,6 +68,17 @@ private class SearchStockMediaDeserializer : JsonDeserializer<SearchStockMediaRe
             media.mediumThumbnail = jsonThumbnails.getString("medium")
             media.postThumbnail = jsonThumbnails.getString("post_thumbnail")
         }
+
+        /**
+         * Guid property needs to be manually parsed to object because comes in String format
```

Comment:
What do you think about rephrasing this a bit? Example : 
```suggestion
         * The GUID property needs to be manually parsed to a JSON object because it was originally deserialized as a string property.
```

Reply:
I agree, your description is way clearer, thanks Joel for the feedback!